### DISCUSSION TITLE
My search function isn’t being called

### DISCUSSION THREAD

**Discussion by ghost** [community/community#58820] [Programming Help] [answered]
### Body

My code is at https://github.com/pytutorials/pytutorials.github.io

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by leonden** [comment, marked as answer]
Can you provide any further information for us to properly understand your issue?

**Comment by ghost** [comment]
Sorry I fixed this error but I forgot to close the discussion.
